### ISSUE TITLE
Missing 'many' suffix for language "fr"

### ISSUE THREAD

**Issue by kncs** [i18next/i18next-parser#460]
## 🐛 Bug Report

The plural suffix 'many' is not generated for fr language. When I generate keys for fr language i only have 'one' and 'other' suffixes generated.

## To Reproduce

I have a react app create with CRA

Launch the command to extract keys 
```sh
i18next --fail-on-warnings 'src/**/*.{js,jsx,ts,tsx}'  -c i18next-parser.config.js
```

```js
// i18next-parser.config.js

module.exports = {
  contextSeparator: '_',
  // Key separator used in your translation keys

  createOldCatalogs: true,
  // Save the \_old files

  defaultNamespace: 'default',
  // Default namespace used in your i18next config

  defaultValue: '##NOTSET##',
  // Default value to give to empty keys
  // You may also specify a function accepting the locale, namespace, and key as arguments

  indentation: 2,
  // Indentation of the catalog files

  keepRemoved: false,
  // Keep keys from the catalog that are no longer in code

  keySeparator: '.',
  // Key separator used in your translation keys
  // If you want to use plain english keys, separators such as `.` and `:` will conflict. You might want to set `keySeparator: false` and `namespaceSeparator: false`. That way, `t('Status: Loading...')` will not think that there are a namespace and three separator dots for instance.

  // see below for more details
  lexers: {
    hbs: ['HandlebarsLexer'],
    handlebars: ['HandlebarsLexer'],

    htm: ['HTMLLexer'],
    html: ['HTMLLexer'],

    mjs: ['JavascriptLexer'],
    js: ['JavascriptLexer'], // if you're writing jsx inside .js files, change this to JsxLexer
    ts: ['JavascriptLexer'],
    jsx: ['JsxLexer'],
    tsx: ['JsxLexer'],

    default: ['JavascriptLexer']
  },

  lineEnding: 'auto',
  // Control the line ending. See options at https://github.com/ryanve/eol

  locales: ['en', 'fr'],
  // An array of the locales in your applications

  namespaceSeparator: ':',
  // Namespace separator used in your translation keys
  // If you want to use plain english keys, separators such as `.` and `:` will conflict. You might want to set `keySeparator: false` and `namespaceSeparator: false`. That way, `t('Status: Loading...')` will not think that there are a namespace and three separator dots for instance.

  output: 'public/locales/$LOCALE/$NAMESPACE.json',
  // Supports $LOCALE and $NAMESPACE injection
  // Supports JSON (.json) and YAML (.yml) file formats
  // Where to write the locale files relative to process.cwd()

  pluralSeparator: '_',
  // Plural separator used in your translation keys
  // If you want to use plain english keys, separators such as `_` might conflict. You might want to set `pluralSeparator` to a different string that does not occur in your keys.

  input: undefined,
  // An array of globs that describe where to look for source files
  // relative to the location of the configuration file

  sort: false,
  // Whether or not to sort the catalog

  skipDefaultValues: false,
  // Whether to ignore default values
  // You may also specify a function accepting the locale and namespace as arguments

  useKeysAsDefaultValue: false,
  // Whether to use the keys as the default value; ex. "Hello": "Hello", "World": "World"
  // This option takes precedence over the `defaultValue` and `skipDefaultValues` options
  // You may also specify a function accepting the locale and namespace as arguments

  verbose: false,
  // Display info about the parsing including some stats

  failOnWarnings: false,
  // Exit with an exit code of 1 on warnings

  customValueTemplate: null
  // If you wish to customize the value output the value as an object, you can set your own format.
  // ${defaultValue} is the default value you set in your translation function.
  // Any other custom property will be automatically extracted.
  //
  // Example:
  // {
  //   message: "${defaultValue}",
  //   description: "${maxLength}", // t('my-key', {maxLength: 150})
  // }
};
```

## Expected behavior

According to this link found into the i18next documentation I was expected the suffix  'many'  will be generated for the fr language.

https://jsfiddle.net/6bpxsgd4

## Your Environment

- *runtime version*:  node 14.16.1
- *i18next version*: i.e. 21.0.0
- *os*: Linux
- *i18next-parser version* : 5.0.0

**Comment by karellm** [comment]
@Elindorath Would you happen to know anything about this?

**Comment by Elindorath** [comment]
Sure!
This behavior depends on the implementation of the Intl API.
As i mentionned [here](https://github.com/i18next/i18next-parser/pull/442#issuecomment-934227389), the `many` suffix for french was introduced in [CLDR v38](https://icu.unicode.org/download/68), included in the [node's ICU dependencies v68.2](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V14.md?rgh-link-date=2021-10-05T09%3A20%3A56Z#other-notable-changes). This update is available from Node v14.17.0.

All this to advise you to update Node 😉

**Comment by karellm** [comment]
Thanks @Elindorath 

@kncs I'm closing this as it doesn't seem linked to this project directly but to node itself.
